pytest_html/plugin.py

```diff
@@ -215,23 +220,55 @@ def pytest_sessionfinish(self, session):
             html.title('Test Report'),
             html.style(raw(style_css)))
 
+        class Outcome:
+
+            def __init__(self, outcome, total=0, label=None,
+                         test_result=None, class_html=None):
+                self.outcome = outcome
+                self.label = label or outcome
+                self.class_html = class_html or outcome
+                self.total = total
+                self.test_result = test_result or outcome
+
+                self.generate_checkbox()
+                self.generate_summary_item()
+
+            def generate_checkbox(self):
+                data_test_result = {"data-test-result":
+                                    self.test_result.lower()}
+                if(self.total > 0):
+                    self.checkbox = html.input(type='checkbox',
+                                               checked='true',
+                                               onChange='filter_table(this)',
+                                               name='filter_checkbox',
+                                               **data_test_result)
+                else:
+                    self.checkbox = html.input(type='checkbox',
+                                               disabled='true')
```

Comment:
Ah, right, it's a boolean attribute that doesn't require a value. You could probably do it by making `data_test_result` more generic, such as:

```python
checkbox_kwargs = {
    'data-test-result': self.test_result.lower()}

if self.total > 0:
    checkbox_kwargs['disabled'] = 'true'

self.checkbox = html.input(
    type='checkbox',
    checked='true',
    onChange='filter_table(this)',
    name='filter_checkbox',
    **checkbox_kwargs)
```